functions/api/v1/instance.ts

```diff
@@ -1,36 +1,99 @@
+// https://docs.joinmastodon.org/entities/Instance/
+// https://docs.joinmastodon.org/methods/instance/
 import type { Env } from 'wildebeest/backend/src/types/env'
 import { cors } from 'wildebeest/backend/src/utils/cors'
+import * as error from 'wildebeest/backend/src/errors'
 import { DEFAULT_THUMBNAIL } from 'wildebeest/backend/src/config'
 import { getVersion } from 'wildebeest/config/versions'
+import { calculateInstanceStatistics } from 'wildebeest/backend/src/mastodon/instance'
+import { MastodonInstance } from 'wildebeest/backend/src/types/instance'
+import { MastodonAccount } from 'wildebeest/backend/src/types/account'
+import { loadLocalMastodonAccount } from 'wildebeest/backend/src/mastodon/account'
+import { Database, getDatabase } from 'wildebeest/backend/src/database'
+import { getAdmins } from 'wildebeest/backend/src/utils/auth/getAdmins'
+import { getRules } from 'wildebeest/backend/src/config/rules'
+import { emailSymbol } from 'wildebeest/backend/src/activitypub/actors'
 
 export const onRequest: PagesFunction<Env, any> = async ({ env, request }) => {
-	const domain = new URL(request.url).hostname
-	return handleRequest(domain, env)
+	const domain: string = new URL(request.url).hostname
+	const db: Database = await getDatabase(env)
+	return handleRequest(domain, env, db)
 }
 
-export async function handleRequest(domain: string, env: Env) {
+export async function handleRequest(domain: string, env: Env, db: Database) {
 	const headers = {
 		...cors(),
 		'content-type': 'application/json; charset=utf-8',
 	}
 
-	const res: any = {}
+	if (env.ADMIN_EMAIL === 'george@test.email') {
+		db = await getDatabase(env)
```

Comment:
why do you need this? db is already passed in the function, do you still need to call getDatabase?